cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -526,6 +526,28 @@ bundle agent cfe_internal_refresh_inventory_view
 
 }
 
+bundle agent cfe_internal_refresh_hosts_view
+# @brief Refresh hosts materialized view every 5 minutes
+{
+  meta:
+
+    (policy_server|am_policy_hub).enterprise_edition::
+
+      "tags" slist => { "enterprise_maintenance" };
+
+  commands:
+
+    (policy_server|am_policy_hub).enterprise_edition::
+
+      "$(sys.workdir)/httpd/php/bin/php"
+        args => "$(cfe_internal_hub_vars.docroot)/index.php cli_tasks materialized_hosts_view_refresh",
+        comment => "This refreshes the hosts view. If the hosts view is not refreshed then it will contain stale data.",
+        handle  => "mpf_fresh_hosts_view",
+        classes => kept_successful_command,
+        if => countlinesmatching("materialized_hosts_view_refresh", "$(cfe_internal_hub_vars.docroot)/application/controllers/Cli_tasks.php");
```

Comment:
I don't think this constraint makes sense.

countlinesmatching() returns an int not a boolean. https://docs.cfengine.com/docs/3.10/reference-functions-countlinesmatching.html

I guess this should probably be something like `if => isgreaterthan( countlinesmatching(), 0 );`